### PR TITLE
SF Symbols 2.0 added ✅

### PR DESCRIPTION
**Pull request by aaryankotharii**
# SF Symbols 2.0 added

- Includes an enum named SFSymbols2 with 800 cases of new symbols.

- isMulticolor computed property added to the enum.

- all extensions are enclosed under iOS14 available condition.

- added respective UIIMage and Image extension to use these.

- [Aaryan Kothari](https://github.com/aaryankotharii)

### CODE REVIEW

File: Sources/SFSymbol/Restricted+SFSymbol.swift [hunk 1]
```diff
@@ -18,4 +18,5 @@ public extension SFSymbol {
             return false
         }
     }
+
```
Comment (Nirma) [inline Sources/SFSymbol/Restricted+SFSymbol.swift]: Good job with the pull request! This change isn't needed right?

File: Sources/SFSymbol/SFSymbol.swift [hunk 2]
```diff
@@ -1666,3 +1666,6 @@ public enum SFSymbol: String, CaseIterable {
     case zSquare = "z.square"
     case zzz = "zzz"
 }
+
```
Comment (Nirma) [inline Sources/SFSymbol/SFSymbol.swift]: This too, delete this if you don't mind.

File: SFSymbol2/SFSymbol2.0.swift [hunk 3]
```diff
@@ -0,0 +1,811 @@
...
+    case arrowtriangleleftfillandlineverticalandarrowtrianglerightfill = "arrowtriangleLeftFillAndLineVerticalAndArrowtriangleRightFill"
+    case arrowtrianglerightandlineverticalandarrowtriangleleft = "arrowtriangleRightAndLineVerticalAndArrowtriangleLeft"
+    case arrowtrianglerightfillandlineverticalandarrowtriangleleftfill = "arrowtriangleRightFillAndLineVerticalAndArrowtriangleLeftFill"
+    case lifepreserver = "lifepreserver"
+    case lifepreserverfill = "lifepreserverFill"
+    case binoculars = "binoculars"
+    case binocularsfill = "binocularsFill"
+    case battery100bolt = "battery100Bolt"
+    case minusplusbatteryblock = "minusPlusBatteryblock"
+    case minusplusbatteryblockfill = "minusPlusBatteryblockFill"
```
Comment (Nirma) [inline SFSymbol2/SFSymbol2.0.swift]: The names of the `case` statements so far use camel case so for example `case minusplusbatteryblockfill = "minusPlusBatteryblockFill"` should be `case minusPlusBatteryBlockFill = "minusPlusBatteryblockFill"`

File: SFSymbol2/SFSymbol2.0.swift [hunk 4]
```diff
@@ -0,0 +1,811 @@
+//
+//  SFSymbol2.0.swift
```
Comment (Nirma) [inline SFSymbol2/SFSymbol2.0.swift]: Good job so far! Just that this file name should be something like `SFSymbol2.swift`

File: SFSymbol2/Image+SFSymbol2.0.swift [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+//  Image+SFSymbol.swift
+//  SFSymbol
+//
+//  Created by Aaryan Kothari on 25/07/20.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+public extension Image {
```
Comment (Nirma) [inline SFSymbol2/Image+SFSymbol2.0.swift]: This is fine!

**Comment by aaryankotharii** [comment]
Ill make the necessary changes 👍🏻.

**Comment by aaryankotharii** [comment]
**Changes made.**

**Comment by Nirma** [comment]
@aaryankotharii Ok LGTM!

**Pull request merged**